packages/viz/src/Boxplot/Boxplot.tsx

```diff
@@ -0,0 +1,132 @@
+import { forwardRef } from "react";
+import ReactECharts from "echarts-for-react/lib/core";
+import { BoxplotChart } from "echarts/charts";
+import { TooltipComponent, VisualMapComponent } from "echarts/components";
+import * as echarts from "echarts/core";
+import { Arrayable, ExtractNames } from "@hitachivantara/uikit-react-core";
+
+import { HvBaseChart } from "../BaseChart";
+import {
+  useGrid,
+  useLegend,
+  useOption,
+  useTooltip,
+  useXAxis,
+  useYAxis,
+} from "../hooks";
+import { HvBoxplotMeasure, HvChartTooltip } from "../types";
+import { HvAxisChartCommonProps, HvChartCommonProps } from "../types/common";
+import { useClasses } from "./Boxplot.styles";
+import { useBoxplot } from "./useBoxplot";
+
+// Register chart components
+echarts.use([BoxplotChart, TooltipComponent, VisualMapComponent]);
+
+export type HvBoxplotClasses = ExtractNames<typeof useClasses>;
+
+export interface HvBoxplotProps
+  extends HvAxisChartCommonProps,
+    HvChartCommonProps {
+  /** The name of the heatmap */
+  name?: string;
+  /**  Columns to measure on the chart. */
+  measures: Arrayable<HvBoxplotMeasure>;
+  /** The tooltip options. */
+  tooltip?: Omit<HvChartTooltip, "type">;
```

Comment:
Is `valueFormatter` and `titleFormatter` used? If not, we should omit these props I think.